src/main/java/de/tum/in/www1/artemis/service/iris/session/IrisChatSessionService.java

```diff
@@ -163,6 +166,16 @@ else if (irisMessage != null) {
                     return null;
                 });
     }
+    
+    private static IrisMessage toIrisMessage(IrisMessageResponseDTO dto) {
+        var message = new IrisMessage();
+        message.setSentAt(dto.sentAt());
+        message.setSender(IrisMessageSender.LLM);
+        var irisMessageContent = new IrisMessageContent();
+        irisMessageContent.setTextContent(dto.content().get("response").asText());
```

Comment:
Have you tested what happens if the "response" attribute does not exist?
As far as I can see this would silently throw an error without notifying the client. If that is indeed so, please add some simple error handling for this, which sends the error over the websocket.